test-bot/src/main/kotlin/com/kotlindiscord/kord/extensions/testbot/TestBot.kt

```diff
@@ -38,6 +43,16 @@ public suspend fun main() {
 
         applicationCommands {
             defaultGuild(TEST_SERVER_ID)
+
+            useLimiter {
+                cooldownHandler = DefaultCooldownHandler()
+                rateLimiter = DefaultRateLimiter()
```

Comment:
Shouldn't these be set by default, if they're the default types?

Reply:
they are set by default, I can remove this if it's too verbose